### PR TITLE
TETR.IO Tetra League fix

### PR DESCRIPTION
**Pull request by magumboi**
image

### CODE REVIEW

**Review by Bas950 — APPROVED** [review]

File: websites/T/TETR.IO/dist/metadata.json [hunk 1]
```diff
@@ -4,18 +4,25 @@
     "name": "Rare Akuma ☆",
     "id": "250044591686811669"
   },
+  "contributors": [
+    {
+      "name": "Hyruled",
+      "id": "600754443318853632"
+    }
+  ],
```
Comment (ririxidev) [inline websites/T/TETR.IO/dist/metadata.json:12]: Didn't see him doing pull request about doing things so delete it.

**Review by ririxidev — CHANGES_REQUESTED** [review]
resolve conflicts too.

File: websites/T/TETR.IO/dist/metadata.json (hunk 1 above)
Comment (magumboi) [reply websites/T/TETR.IO/dist/metadata.json:12]: oh, I see, np

**Review by Bas950 — APPROVED** [review]

File: websites/T/TETR.IO/dist/metadata.json [hunk 2]
```diff
@@ -9,13 +9,14 @@
   "description": {
     "en": "TETR.IO is a free-to-play online multiplayer and single player fan game"
   },
-  "version": "1.0.1",
+  "version": "1.1.0",
```
Comment (StrikerFRFX) [inline websites/T/TETR.IO/dist/metadata.json:12]: Why bumped to 1.1.0? You could just bump it to 1.0.2 because it's a minor fix...

**Review by StrikerFRFX — APPROVED** [review]

**Pull request merged**